runtest.py

```diff
@@ -88,10 +90,17 @@ def clean_up_output_data(self):
             shutil.rmtree(output_dir)
 
     def _load_settings(self):
+        if not self.firefox_settings:
+            return {}
         try:
             if os.path.exists(self.firefox_settings):
                 with open(self.firefox_settings) as firefox_settings_fh:
-                    return json.load(firefox_settings_fh)
+                    settings = json.load(firefox_settings_fh)
+                    self.logger.info('** Important **\n'
```

Comment:
Try to use a larger warning slogan like:


           *****************************
           *                WARNING               *
           *****************************
THIS WILL OVERRIDE YOUR CURRENT FIREFOX SETTINGS.
EXIT NOW IF YOU DON'T WISH TO DO SO.


I believe people could see this more easier.